tools/rosbag/src/record.cpp

```diff
@@ -145,8 +145,12 @@ rosbag::RecorderOptions parseOptions(int argc, char** argv) {
         if (freq == 0)
         {
             opts.custom_record_freq.emplace(it->first.as<std::string>(), ros::Duration(-1));
-        } 
-        else 
+        }
+        else if (freq == -1)
+        {
+            opts.custom_record_freq.emplace(it->first.as<std::string>(), ros::Duration(-2));
```

Comment:
shouldn't this just be `ros::Duration(0)` without need for special handling?

Reply:
Yes, that can be done, will test.